File: v18/src/app/shared/models/blogs/tag.ts
import { BlogPost } from './blogPost';

export interface Tag {
  id: string;
  name: string;
  displayName: string;
  blogPosts: BlogPost[];
}

export interface EditTag {
  id?: string;
  name: string;
  displayName: string;
}

export interface Tags {
  id: string;
  name: string;
  displayName: string;
  count: number;
}

File: API/Models/Blog/Tag.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace API.Models.Blog
{
    public class Tag
    {
        //public Guid Id { get; set; }
        public long Id { get; set; }
        public string Name { get; set; }
        public string DisplayName { get; set; }        
        //public ICollection<BlogPost> BlogPosts { get; set; } = new List<BlogPost>();
    }
}
